packages/api/src/modules/providers/datagouv/historyUniteLegal/adapters/UniteLegaleHistoriqueAdapter.ts

```diff
@@ -8,12 +8,12 @@ export class UniteLegaleHistoriqueAdapter {
         return new UniteLegalNameEntity(
             row.siren,
             row.denominationUniteLegale,
-            this.buildSearchingKey(row.siren, row.denominationUniteLegale),
+            this.buildsearchKey(row.siren, row.denominationUniteLegale),
             new Date(row.dateDebut),
         );
     }
 
-    private static buildSearchingKey(siren: Siren, name: string) {
+    private static buildsearchKey(siren: Siren, name: string) {
```

Comment:
```suggestion
    private static buildSearchKey(siren: Siren, name: string) {
```
presque, tu as pas mis l'option "conserver la casse" en remplaçant

Reply:
Fichtre

Reply:
ça push